### DISCUSSION TITLE
Can I upload an image?

### DISCUSSION THREAD

**Discussion by topdock** [ocornut/imgui#6522] [New Users Build/Link/Run issues ONLY!]
Can I upload an image from the Font-Awesome site and import it from a personal project?

**Comment by PathogenDavid** [comment]
It's not really clear what you're asking, but it seems unrelated to Dear ImGui. [Posting](https://github.com/FortAwesome/Font-Awesome/discussions/19763) the [same](https://github.com/dyanikoglu/ALS-Community/discussions/453) vague question to multiple repositories is not a good way to get an answer.

If you're trying to ask about incorporating your own images into the Font Awesome icon set, my understanding is that is a [paid feature](https://fontawesome.com/plans).

If you're using Dear ImGui you don't actually need to have the icon embedded in a font, you can just add arbitrary images to the font. See [the custom icons documentation](https://github.com/ocornut/imgui/blob/194916135ad34b92bf6aea1e76dfcab0ea9924c8/docs/FONTS.md#using-custom-colorful-icons) for details.
